test/extensions/transport_sockets/http_11_proxy/connect_test.cc

```diff
@@ -117,6 +119,30 @@ class Http11ConnectTest : public testing::TestWithParam<Network::Address::IpVers
         anypb.PackFrom(addr_proto);
         metadata->mutable_typed_filter_metadata()->emplace(std::make_pair(metadata_key, anypb));
         EXPECT_CALL(*host, metadata()).Times(AnyNumber()).WillRepeatedly(Return(metadata));
+
+        if (with_hostname) {
+          static const std::string hostname = "test.example.com";
+          EXPECT_CALL(*host, hostname()).Times(AnyNumber()).WillRepeatedly(ReturnRef(hostname));
+
+          // Create real address with port 443
+          static auto real_address =
```

Comment:
`static` isn't necessary as it will keep the object from being freed for the lifetime of the process. It's a test so it shouldn't really matter, but let's remove it anyway.